pulp_ansible/app/tasks/collectionversion_index.py

```diff
@@ -175,7 +183,11 @@ def update_index(distribution=None, repository=None, repository_version=None, is
     for colkey in colset:
         namespace, name = colkey
 
-        if changed_collections is not None and (namespace, name) not in changed_collections:
+        if (
+            changed_collections is not None
+            and (namespace, name) not in changed_collections
+            and (namespace, "*") not in changed_collections
+        ):
```

Comment:
I'm not sure what circumstance would make `changed_collections` be `None`, but  (prior to this PR and also with this PR) this logic seems to do this:

if `changed_collections` is `None`, it will not `continue`, so it will attempt the rest of the create|update index... is that desired?

Reply:
That is correct. However changed_collections reflects changes to CVs, deprecatation, signatures and now namespacemetadata.

Reply:
```suggestion
        if changed_collections is None:
            continue
        if (
            (namespace, name) not in changed_collections
            and (namespace, "*") not in changed_collections
        ):
```

IIUC, and this is logically correct, this is a suggestion to be explicty that there is no indexing when no changed collections, optional edit

Reply:
probably better to just remove check for `changed_collections is not None`